libs/server-sdk/README.md

```diff
@@ -102,6 +102,34 @@ for consistency across SDKs, as well as test networking behavior in a long-runni
 method in the SDK, and verify that event sending, flag evaluation, stream reconnection, and other aspects of the SDK all
 behave correctly.
 
+Validating SDK packages with the SLSA framework
+-------
+
+LaunchDarkly uses the [SLSA framework](https://slsa.dev/spec/v1.0/about) to help developers make their supply chain more secure by ensuring the authenticity of our published SDK packages. As part of [SLSA requirements for level 3 compliance](https://slsa.dev/spec/v1.0/requirements), LaunchDarkly publishes provenance about our SDK package builds using [Github's generic SLSA3 provenance generator](https://github.com/slsa-framework/slsa-github-generator/blob/main/internal/builders/generic/README.md#generation-of-slsa3-provenance-for-arbitrary-projects) for distribution alongside our packages. 
+
+The SLSA framework specifies some [recommendations for verifying build artifacts](https://slsa.dev/spec/v1.0/verifying-artifacts) in their documentation. For our C/C++ SDKs, we recommend the following steps:
+- From the Github releases page for the release version, download the provenance file for the OS you're developing on (`OSNAME-multiple-provenance.intoto.jsonl`)
+- Check the provenance payload (`payload` in `OSNAME-multiple-provenance.intoto.jsonl`) to verify the authenticity of the build:
+  - The payload is wrapped in a DSEE envelope in encoded in base64. To make the provenance human readable, base64 decode the `payload` parameter, e.g.: 
+    ```cat OSNAME-multiple-provenance.intoto.jsonl | jq -r '.payload' | base64 -d | jq```
+  - Validate that the subject digests in `payload.subject` match the sha256 checksums of the SDK packages for the OS you're developing in
```

Comment:
How to do this step? What files exactly should they check, and using what tool? (probably `sha256`)